.rubocop.yml

```diff
@@ -27,6 +27,8 @@ Naming/FileName:
     - lib/flatware-*.rb
 Style/Documentation:
   Enabled: false
+Metrics/AbcSize:
+  Max: 20
```

Comment:
This change has added a bit of extra AbcSize so RUbocop started failing.

https://github.com/briandunn/flatware/blob/ec406db69e030bc03eb163ed027b564dae2fb23e/lib/flatware/rspec/cli.rb#L25-L26

I looked to see if there is any meaningful refactoring opportunity here, but even a few deliberate changes (e.g. just moving lines around, extracting and putting it in a different file, etc) didn't get the Rubocop check passing. I don't think I can come up with meaningfully refactored code out of this low AbsSize setting, so I just increased the size.

I know people have opinions about this, but this is coming a hurdle to new contributors...